src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -612,7 +609,7 @@ private enum AssignmentKind
         /// <summary>
         /// Reports top-level nullability problem in assignment.
         /// </summary>
-        private bool ReportNullableAssignmentIfNecessary(BoundExpression value, TypeSymbolWithAnnotations targetType, TypeSymbolWithAnnotations valueType, bool useLegacyWarnings, AssignmentKind assignmentKind = AssignmentKind.Assignment, Symbol target = null)
+        private bool ReportNullableAssignmentIfNecessary(BoundExpression value, TypeSymbolWithAnnotations valueType, TypeSymbolWithAnnotations targetType, bool useLegacyWarnings, AssignmentKind assignmentKind = AssignmentKind.Assignment, Symbol target = null)
```

Comment:
>TypeSymbolWithAnnotations valueType, TypeSymbolWithAnnotations targetType [](start = 80, length = 73)

Why switch the order?

Reply:
Explained elsewhere: to put `value` and `valueType` together. The previous order of parameters regularly tripped me up.

---
In reply to: [258133264](https://github.com/dotnet/roslyn/pull/33447#discussion_r258133264) [](ancestors = 258133264)

Reply:
I'll do the re-ordering in a separate PR after Neal merges his big change, to minimize conflicts.

---
In reply to: [258229068](https://github.com/dotnet/roslyn/pull/33447#discussion_r258229068) [](ancestors = 258229068,258133264)